modules/doc/prelude-cheatsheet.tex

```diff
@@ -12,15 +12,13 @@
 \usepackage{calc}
 \usepackage{tcolorbox}
 
-\pagecolor{black}
+\usepackage{style-dark}
```

Comment:
Oh, I didn't notice this. Seems to me we can simply change the default.

Reply:
So if I understand correctly, your suggestion is to only have one version of the cheatsheet - the one with black text on white background? I personally don't care about the original dark version, but I think it's nice if the users can choose between the light or dark version according to their liking.

Reply:
Let's not overdo this and have just a single version. :-) 

I don't think it makes sense to make customizable something as basic as this, that's probably used by about 10 people. :-)

Reply:
Ok, if we're not married to the dark version as you say, then I will make a new PR with just the colors adjusted. I'm closing this PR now. Thanks for your advice.